core/config.go

```diff
@@ -53,6 +53,25 @@ func (b BodyConfig) CreatePrompt(stdinReader io.ReadCloser) Prompt {
 	return p
 }
 
+type NewLineConfig struct {
```

Comment:
small capitalization change: 
```suggestion
type NewlineConfig struct {
```